Source/Modules/java.cxx

```diff
@@ -2731,10 +2731,10 @@ class JAVA:public Language {
       // Default annotation for director constructors is a warning suppression
       static const String *suppress_warning_this_escape = NewString("@SuppressWarnings(\"this-escape\")");
       if (!annotations && feature_director)
-	annotations = suppress_warning_this_escape;
+	  annotations = suppress_warning_this_escape;
       if (annotations) {
-	Printf(function_code, "  %s\n", annotations);
-	Printf(helper_code, "  %s\n", annotations);
+	    Printf(function_code, "  %s\n", annotations);
+	    Printf(helper_code, "  %s\n", annotations);
       }
```

Comment:
Please don't make changes to re-indent code entangled with functional changes.  Especially not code that's unrelated to the functional changes.

In this case the indentation looks correct as-is, so this change is also unwanted anyway (maybe you have your tab width wrongly set - it should move to the next multiple of 8 columns for SWIG code).

If the indentation actually is wrong please open a separate PR for fixing it.